policyengine_us/variables/gov/states/mt/dhs/tanf/income/earned/mt_tanf_earned_income_after_disregard_person.py

```diff
@@ -0,0 +1,28 @@
+from policyengine_us.model_api import *
+
+
+class mt_tanf_earned_income_after_disregard_person(Variable):
+    value_type = float
+    entity = Person
+    label = "MT Temporary Assistance for Needy Families (TANF) earned income after disregard per person"
```

Comment:
```suggestion
    label = "Montana Temporary Assistance for Needy Families (TANF) earned income after disregard per person"
```

We typically write out the state name for easier parsing the web app